ConfigApp/MainWindow.xaml.cs

```diff
@@ -17,7 +17,7 @@ public partial class MainWindow : Window
         private bool m_initializedTitle = false;
 
         private OptionsFile m_configFile = new OptionsFile("config.ini");
-        private OptionsFile m_twitchFile = new OptionsFile("twitch.ini");
+        private OptionsFile m_streamFile = new OptionsFile("voting.ini");

```

Comment:
So the only reason we may _not_ want to rename this is to have a seamless upgrade. Or at least, have some "if twitch.ini exists, rename it to voting.ini" condition

Reply:
Issue with a rename is some of the config option names were also changed, I figured it automatically renaming would make it potentially confuse people with old setups, when the file is renamed, but not working, then a clear, recreate needed